### PR TITLE
Instructions on how to deploy your copilot to Fly.io (ENG-371)

### PR DESCRIPTION
**Pull request by taivokasper**
### Task & problem
Task: [How to deploy a copilot easily?](https://linear.app/nftport/issue/ENG-371/how-to-deploy-a-copilot-easily)
Solves the problem of how to share your creation with the world (or friends)

### Changes made
Added an example copilot which includes deployment instructions in README.md

### CODE REVIEW

File: opencopilot/html/opencopilot.js
Comment (taivokasper) [inline opencopilot/html/opencopilot.js:1]: Delete this file once React SDK is published

Comment (taivokasper) [inline opencopilot/html/opencopilot.js:1]: Delete this file once React SDK is published

File: examples/flyio_deployed_copilot/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+opencopilot-ai>=0.3.1
```
Comment (taivokasper) [inline examples/flyio_deployed_copilot/requirements.txt:1]: Increase the version to include the weaviate_api_key changes in [weaviate pr](https://github.com/opencopilotdev/opencopilot/pull/74)

**Comment by taivop** [comment]
Closing as now the recommended deployment is with repl.it.

**Pull request closed without merging**